Common/UI/Context.cpp

```diff
@@ -83,8 +83,6 @@ void UIContext::BeginPipeline(Draw::Pipeline *pipeline, Draw::SamplerState *samp
 	// Also clear out any other textures bound.
 	Draw::SamplerState *samplers[3]{ samplerState };
 	draw_->BindSamplerStates(0, 3, samplers);
-	Draw::Texture *textures[2]{};
-	draw_->BindTextures(1, 2, textures);
```

Comment:
Hm, what was the reason for removing these?  I feel like we've hit errors and problems when bad things are left bound, so I think this was intended to wipe it.  Note that this is clearing slots 1 and 2, and RebindTexture() is setting slot 0.

Maybe it's not needed on every BeginPipeline or something, but I'm assuming it was added to fix some error...

-[Unknown]

Reply:
Oh I had some *wacky* bugs initially by using the wrong offset in the descriptor writer, this was meant to be a temporary, oops..